package.json

```diff
@@ -17,9 +19,8 @@
     "p-map": "~1.1.1"
   },
   "peerDependencies": {
-    "orbit-db-store": "4.x"
+    "orbit-db-store": "github:orbitdb/orbit-db-store#esm"
```

Comment:
This should be "next" (and prolly want to update package-lock.json after doing so).